src/log.h

```diff
@@ -3,12 +3,11 @@
 
 #include <string>
 
-extern int LOG_LEVEL;
-
 constexpr int LOG_TRACE = 0;
 constexpr int LOG_DEBUG = 1;
 constexpr int LOG_INFO = 2;
 constexpr int LOG_WARN = 3;
 constexpr int LOG_ERROR = 4;
 
 void log(int level, const char* fmt, ...);
+void setLogLevel(int level);
```

Comment:
I know it wasn't there before, but could we add doxygen documentation to these methods?